catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -345,34 +336,38 @@ const WorkDetails = ({
           )}
         </WorkDetailsSection>
 
-        {licenseInfo && (
-          <WorkDetailsSection headingText="License information">
-            <div id="licenseInformation">
-              {licenseInfo.humanReadableText.length > 0 && (
+        {licenseInfo.length > 0 &&
```

Comment:
Do you need the `licenseInfo.length > 0` here? If the length is zero, then the map won't output anything anyhow.